secure-vespa-with-mtls/README.md

```diff
@@ -33,26 +37,26 @@ $ $VESPA_SAMPLE_APP/scripts/generate_hosts_xml.sh | tee $VESPA_SAMPLE_APP/src/ma
 </pre>
 **Wait for the configuration server to start (should return 200 OK):**
 <pre data-test="exec" data-test-wait-for="200 OK">
-$ curl -s --head $(hostname):19071/ApplicationStatus
+$ $VESPA_SAMPLE_APP/scripts/vespa-curl.sh -s --head https://localhost:19071/ApplicationStatus
 </pre>
 **Deploy the application:**
 <pre data-test="exec">
 $ $VESPA_SAMPLE_APP/scripts/deploy.sh
 </pre>
 **Wait for the application to start (should return 200 OK):**
 <pre data-test="exec" data-test-wait-for="200 OK">
```

Comment:
Note: `data-test-wait-for` is incorrect if curl uses *HTTP/2* - which occurs for any Vespa release since jdisc got *HTTP/2* by default. We can resolve this by adding `--http1.1` to curl.